packages/elements/src/calendar/index.ts

```diff
@@ -800,7 +801,29 @@ export class Calendar extends ControlElement implements MultiValue {
   }
 
   /**
-   * Run when tap event happens ot table.
+   * get parent of node that contain value
+   * @param event Keyboard event
+   * @returns html element
+   */
+  private getCellParent(event: KeyboardEvent) {
```

Comment:
I think add return types would be better.

Reply:
fixed